### PR TITLE
DIV-4420: minipetition draft endpoints added to prod

### PR DESCRIPTION
**Pull request by yarpo**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseEvent.json [hunk 1]
```diff
@@ -1068,6 +1070,7 @@
     "PostConditionState": "SOTAgreementPayAndSubmitRequired",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-create",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/solicitor-created",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent.json]: this was added by @alexwakeman on 04/07. 
commit 1f22efbd08dfccc8708d40d15cc84dc6641b20bb

I don't know if we actually need both urls as they are the same...

Comment (yarpo) [reply definitions/divorce/json/CaseEvent.json]: @hosainnet do you think this ^^ is ok?

Comment (tchow8) [reply definitions/divorce/json/CaseEvent.json]: One is -create and the other is -created (the notifications enpdoint) I think?

Comment (tchow8) [reply definitions/divorce/json/CaseEvent.json]: I don't think its merged yet, https://github.com/hmcts/div-case-orchestration-service/pull/357

Comment (yarpo) [reply definitions/divorce/json/CaseEvent.json]: Look at master, it's in:
https://github.com/hmcts/div-ccd-definitions/blob/master/definitions/divorce/json/CaseEvent.json#L1075

Shall I add it to prod in my PR or remove this line as the feature seems not to be prod-ready.

Comment (yarpo) [reply definitions/divorce/json/CaseEvent.json]: I will remove it from this PR

File: definitions/divorce/json/CaseType.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
-    "Name": "Divorce case - v114.13",
+    "Name": "Divorce case - v114.18",
```
Comment (hosainnet) [inline definitions/divorce/json/CaseType.json]: Big jump here?

Comment (yarpo) [reply definitions/divorce/json/CaseType.json]: v114.14

**Pull request merged**